packages/modular-scripts/src/utils/isInWorkspaces.ts

```diff
@@ -0,0 +1,29 @@
+import fs from 'fs-extra';
+import path from 'path';
+import minimatch from 'minimatch';
+import getModularRoot from './getModularRoot';
+import type { ModularPackageJson } from '@modular-scripts/modular-types';
+
+// This checks if a path is included in the workspaces paths of a monorepo
+export async function isInWorkspaces(targetPath: string): Promise<boolean> {
+  const workspaceGlobs = await getWorkspaceGlobs();
+  return workspaceGlobs.some((glob) => minimatch(targetPath, glob));
+}
+
+// This function gets all the globs in "workspaces" field in the root package manifest
+export async function getWorkspaceGlobs(): Promise<string[]> {
```

Comment:
As this module exports 2 different workspace-related functions, perhaps it would better be called `workspaces` (i.e. this is where all utils for workspaces are placed).

Reply:
The second export isn't currently used in the project, so this could be private to the module

Reply:
Removed the `export`